websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -60,7 +60,7 @@ presence.on("UpdateData", async () => {
     else {
       const episode = JSON.parse(
         document.querySelector(
-          "#root > div > div > div.sc-pkSvE.kPCOPp > div > div > div.sc-psOyd.fIwdpb > script"
+          "#root > div > div > div.sc-qPmLO.gspUft > div > div > div.sc-qXiOn.iVbXhX > script"
```

Comment:
Shouldnt you use nth:child or something that is more against the class name changes.